tictactoe/src/components/Board.js

```diff
@@ -1,16 +1,70 @@
-import React from "react";
+import React, { useEffect } from "react";
 import Square from "./Square";
 import "./Board.css";
+import { calculateWinner } from "../helpers/calculateWinner";
 
-export default function Board({ playerMovePositions }) {
-  //console.log(playerMovePositions);
+export default function Board({
+  playerMovePositions,
+  currentPlayer,
+  setCurrentPlayer,
+  setPlayerMovePosition,
+  isOver,
+  setGameOver,
+}) {
+  useEffect(() => {
```

Comment:
The useEffect for the winner checking was a nice touch. 
Clean out the console log

Reply:
Yea, lemmi get on that. Thanks man.